packages/neuron-wallet/src/services/hardware-wallet.ts

```diff
@@ -0,0 +1,54 @@
+import { UnsupportedManufacturer } from 'exceptions'
+import { Hardware, DeviceInfo, Manufacturer, HardwareClass } from './hardware'
+import Ledger from './hardware/ledger'
+
+export default class HardwareWalletService {
```

Comment:
What is the relationship between `HardwareWalletService`, `WalletsService` and `services/hardware`. Would it make sense to move this hardware related class into the `services/hardware` folder?

Reply:
`HardwareWalletService`is indeed a confusing name, it should be named `HardwareService` and moved to `services/hardware`.